### DISCUSSION TITLE
[Bug] 似乎是跨域問題造成無法用ddns的域名訪問docker容器

### DISCUSSION THREAD

**Discussion by lpyedge** [lobehub/lobehub#3742] [Self Hosting]
### 📦 Environment

Docker

### 📌 Version

v1.15.6

### 💻 Operating System

macOS

### 🌐 Browser

Edge, Safari

### 🐛 Bug Description

似乎是跨域問題造成無法用ddns的域名訪問docker容器
服務器是路由器做的DMZ直接暴露公網IP,DDNS做的域名映射.
同服務器上其他容器都可以用域名訪問,但是只有lobechat不可以,
用局域網IP+端口號或者公網IP+段口號都可以訪問.

[a comment by an automated account is not shown]

**Comment by mujiannan** [comment]
我也是群晖容器+webstation+自购域名+ddns，正常使用，无访问异常。

**Reply by BiFangKNT** [reply]
你也是本地部署的logto吗？请问相关的docker-compose和lobe的.env配置文件的相关配置是怎样的？有什么配置要点吗？

**Reply by mujiannan** [reply]
> 你也是本地部署的logto吗？请问相关的docker-compose和lobe的.env配置文件的相关配置是怎样的？有什么配置要点吗？

我用的是群晖的官方套件`SSO-Server`

**Reply by BiFangKNT** [reply]
> > 你也是本地部署的logto吗？请问相关的docker-compose和lobe的.env配置文件的相关配置是怎样的？有什么配置要点吗？
> 
> 我用的是群晖的官方套件`SSO-Server`

这个是本地部署的还是会用到云端服务器？

**Comment by BrandonStudio** [comment]
“无法访问”是指什么？
页面无法加载？消息无法发送？
